language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/SortingUtil.java

```diff
@@ -148,10 +151,10 @@ public static String genSortTextForModule(BallerinaCompletionContext context, LS
     }
 
     /**
-     * Get the sort text for a given completion item within a context where the type descriptors are allowed. 
-     * This assume that the user get the particular completion item by calling 
+     * Get the sort text for a given completion item within a context where the type descriptors are allowed.
+     * This assume that the user get the particular completion item by calling
```

Comment:
Shall we rewrite this as "This **assumes** that the user **gets** the particular completion item by calling"

Reply:
Done